benchmark/benchmark.cpp

```diff
@@ -30,53 +33,85 @@
- * - Macro Statements         - number of statements parsed in macro files (includes multiple uses of the same macro)
- * - Copy Def Statements      - number of statements defined in copy files (only the first occurrence of the copy file)
- * - Macro Def Statements     - number of statements defined in macro files (only the first occurrence of the macro)
- * - Lookahead Statements     - number of statements processed in lookahead mode
- * - Reparsed Statements      - number of statements that were reparsed later (e.g. model statements)
- * - Continued Statements     - number of statements that were continued (multiple continuations of one statement count
- *as one continued statement)
- * - Non-continued Statements - number of statements that were not continued
- * - Lines                    - total number of lines
+ * - File                     - File name
+ * - Success                  - Parsing result
+ * - Errors                   - Number of errors encountered during the parsing
+ * - Warnings                 - Number of warnings encountered during the parsing
+ * - Wall Time                - Parsing duration, wall time
+ * - CPU Time                 - Parsing duration, CPU time
+ * - Executed statements      - Number of executed statements
  * - ExecStatement/ms         - ExecStatements includes open code, macro, copy, lookahead and reparsed statements
- * - Line/ms
- * - Files                    - total number of parsed files
+ * - Line/ms                  - Processed lines per ms
+ * - Top messages             - Top message code with highest occurrence
+ * - Open Code Statements     - Number of statements parsed in open code
+ * - Copy Statements          - Number of statements parsed in copy files (includes multiple uses of the same copy file)
+ * - Macro Statements         - Number of statements parsed in macro files (includes multiple uses of the same macro)
+ * - Copy Def Statements      - Number of statements defined in copy files (only the first occurrence of the copy file)
+ * - Macro Def Statements     - Number of statements defined in macro files (only the first occurrence of the macro)
+ * - Lookahead Statements     - Number of statements processed in lookahead mode
+ * - Reparsed Statements      - Number of statements that were reparsed later (e.g. model statements)
+ * - Continued Statements     - Number of statements that were continued (multiple continuations of one statement count
+ *as one continued statement)
+ * - Non-continued Statements - Number of statements that were not continued
+ * - Lines                    - Total number of lines
+ * - Files                    - Total number of parsed files
  */
 
 using namespace hlasm_plugin;
 
 using json = nlohmann::json;
 
 namespace {
+template<typename T>
+void log_(bool flush, T text)
+{
+    flush ? std::clog << text << std::endl : std::clog << text;
```

Comment:
Should this be something like
```suggestion
std::clog << text << '\n';
if (flush)
    std::clog << std::flush; // or std::clog.flush();
```
?
Or maybe it is not exactly `flush`?